### DISCUSSION TITLE
Download extremely slow after updating 8.2.0.23

### DISCUSSION THREAD

**Discussion by shina-sato** [aclap-dev/video-downloadhelper#107] [Help and Q&A]
### Your question

Today, a prompt to update to 8.2.0.23. 

After updating, download speed for really slow.

One hour program, used to take about less than 10 minutes, takes close to 3 hours.

The site I downloaded is 

https://forjoytv.com/forjoytv#

But I’m afraid you have to subscribe to view the files.

I found someone else experiences a similar slowing problem. But I’m afraid this doesn’t help me much.

https://github.com/aclap-dev/video-downloadhelper/discussions/55

Thanks for your time.

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/satoru",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-9-g63b936e978"
}

Download page information

bitrate
1125000
bulk
undefined
chunked
hls
core_media

{...}
created
1708049444331
descrPrefix
HLS streaming
duration
3297.974
extension
mp4
group
grp-7fce3f03
headers

[...]
id
hls:1f71c3db
isPrivate
false
masterManifest
http://live.forjoytv.com:9083/query/wtkmHz1XU-dOl-so_i2LJlfXK5W6PlgBnrqwZJtN8cQ=.m3u8?type=vod&__cross_domain_user=tU5bhLpXfQMbHrALUjkElJWsq03wlvzPO6o9p51S9Tmhy9mQrGO1OUB08ZGkjj%2FiYIt2sqSCuwE1wbTjcfTh6nOHSZCdOV5DafBWoBLx7Xmm%2F1JAEv1BBUuPtLAD%2Fs5tvdyC1dZ1CTHTeMOmgxLs4EwCS5%2B%2FKkbp1CH7lOw0YVVjGZXBvWyigk0j%2F6lU03P0TrWNTeIqs9LFU0PPP8DQ24apPS0SqbYkKs7CMEZzr2PY53PK2ueqPHmB9Es9OIrP%2FwEnUVoY0ujHaUEwZwwwOdR%2FT7BOS4E40DI84AwZ6VhlotOxY%2FHghbotNXn1O95Mrhh9rK0l3SDqJA%2Bvk0iGPZHaOp%2FQYVXRPclMpSU0ihnFNk6YXwDBtT8t8YYYuJHCMZTI%2F2df0RM977WCc8bFlB3Quc3rVNg8hWSFFoXxt%2BhI1rYONVjKD37QSaNT8ADM%2B4eHSrEyh4FJcCFj1%2FttTMrx9swGmrlN%2FvABN6NZ41nykKZFrfdH8LmIV00oL1no82j58%2BbRSTwCoRZ3krQlPBwiiDeTBcTIXMYrW1p0fq4iGo1J6a8Ky2QB26O4LuQE81VFtb74EQy4jZZIVtEWF7V375yTZW7q7Gr2QeR0RKQ%3D
mediaManifest
http://64.32.31.173:9083/vod/sg1-s12/mb/oD/mboDIS-fZMl7otYd6e1NYg==/video.fpvsegments.m3u8?sc_tk=AZhDKhFoiZi%2B1qLURVxn0WpEyrh3HBPPLZiB%2Bl%2Fnoc9AJgkK3SLnI5f3dZ08i9PxynbFOK7iBopscJJsTddfZpVFfNYcnAHgVBQOR00BNi0%3D&filekey=6LK6cuqgCsONxiuGy9dQVfLKPn9mGkP0EmhNYFQBLyI%3D
opStartDate
1708049456390
operation
downloading
originalId
network-probe:3c3782ac
pageUrl
http://play.forjoytv.com/tvplayer_h5/index.html#reload
raw_bitrate
1554213
referrer
http://play.forjoytv.com/tvplayer_h5/index.html#reload
running
1
status
running
tabId
4
thumbnailUrl

title
Watch Japanese TV Online - forjoytv.com
topUrl
http://play.forjoytv.com/tvplayer_h5/index.html#reload

**Comment by paulrouget** [comment]
Thank you for the detailed bug report. I'll look into it soon.

**Comment by paulrouget** [comment]
Can you test this and tell me if it improves the situation: https://github.com/aclap-dev/video-downloadhelper/discussions/199
